src/platform/packages/shared/kbn-lens-embeddable-utils/config_builder/schema/color.ts

```diff
@@ -9,124 +9,92 @@
 
 import type { TypeOf } from '@kbn/config-schema';
 import { schema } from '@kbn/config-schema';
+import { isNil } from 'lodash';
 import { serializedValueSchema } from './serializedValue';
 
-const colorByValueBase = schema.object({
-  type: schema.literal('dynamic'), // Specifies that the color assignment is dynamic (by value). Possible value: 'dynamic'
+const colorByValueStepsSchema = schema.arrayOf(
+  schema.object({
+    /**
+     * The value from which this color applies (inclusive).
+     */
+    from: schema.maybe(
+      schema.nullable(
```

Comment:
I'm wondering why both undefined or nullable? can we just set to one or the other type of non-defined?